samples/apps/spring-petclinic/.patch/pom.xml.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/pom.xml b/pom.xml
+index ae02829..ef52b15 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -30,6 +30,8 @@
+     <nohttp-checkstyle.version>0.0.4.RELEASE</nohttp-checkstyle.version>
+     <spring-format.version>0.0.27</spring-format.version>
+ 
++    <spring-data-jdbc.version>1.2.1.RELEASE</spring-data-jdbc.version>
++
+   </properties>
+ 
+   <dependencies>
+@@ -46,6 +48,10 @@
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-data-jpa</artifactId>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.boot</groupId>
++      <artifactId>spring-boot-starter-jdbc</artifactId>
++    </dependency>
```

Comment:
I thought that we need only to add `spring-cloud-bindings` lib? Where do we need this?

Reply:
you are right, when I tried the things around I used `spring-data-jpa` and that required additional jdbc dependencies... but it now works without it... no need for these dependencies - removed. Thx.